### ISSUE TITLE
second call to add-rnaquantification fails with db access error

### ISSUE THREAD

**Issue by kozbo** [ga4gh/server#1430]
A script such as this will fail on the second call:
```
ga4gh-admin@ga4gh-demo:/srv/ga4gh0_6_0a8pre$ cat add-rna-qs.sh 
#!/bin/bash
# hg100  hg101  102  hg103  hg104  hg96  hg99
time python repo_dev.py add-rnaquantification -n HG00100 -d "Kallisto quantification info for HG00100" -t /run/shm/1000Genomes/rna_quantifications/1kg-rna.db /run/shm/1000Genomes/rna_quantifications/hg100/abundance.tsv kallisto repo.db 1kgenomes gencode_v24lift37 HG00100

time python repo_dev.py add-rnaquantification -n HG00101 -d "Kallisto quantification info for HG00101" -t /run/shm/1000Genomes/rna_quantifications/1kg-rna.db /run/shm/1000Genomes/rna_quantifications/hg101/abundance.tsv kallisto repo.db 1kgenomes gencode_v24lift37 HG00101
```

**Comment by saupchurch** [comment]
running a similar script locally completes with no access issues - exploring the options did uncover a problem with default naming causing all the quantifications to have the same name as thus fail on the uniqueness check

**Comment by saupchurch** [comment]
fixed in #1435
